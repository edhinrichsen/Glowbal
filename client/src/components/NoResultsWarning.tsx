import React from 'react';
import { Result } from 'antd';

import './NoResultsWarning.css';

function NoResultsWarning() {
  return (
    <div className="no-results-warning">
      <Result status="warning" title="No results found" />
    </div>
  );
}

export default NoResultsWarning;
