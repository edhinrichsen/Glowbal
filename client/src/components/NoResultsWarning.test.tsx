import React from 'react';
import { shallow } from 'enzyme';
import NoResultsWarning from './NoResultsWarning';

describe('Testing NoResultsWarning', () => {
  it('renders without crashing', () => {
    const component = shallow(<NoResultsWarning />);
    expect(component).toMatchSnapshot();
    component.unmount();
  });
});
