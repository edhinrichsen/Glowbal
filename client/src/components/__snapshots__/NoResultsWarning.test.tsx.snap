// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing NoResultsWarning renders without crashing 1`] = `
<div
  className="no-results-warning"
>
  <Result
    status="warning"
    title="No results found"
  />
</div>
`;
